### ISSUE TITLE
HasNotifier implementation for events

### ISSUE THREAD

**Issue by mar-ar** [FreeOpcUa/opcua-asyncio#105]
Hi guys,

I realized that the implementation of the events isn´t complete at the moment.
According to the specs each event should be received if you subscribe to the server node.
At the moment there is no way to implement an event hierarchy.
Here is a event hierarchy from the specs:

![image](https://user-images.githubusercontent.com/38347149/69862917-d0c4a300-129b-11ea-8956-97d14bf3199a.png)

I will try to implement such a behavior.

**Comment by oroulet** [comment]
There are many things in the spec which is not implemented by any server or clients ;-)
Or server supports events coming from any node. This is already rare (at least it was)
But yes if you find missing stuff, you are welcome to implement them

**Comment by mar-ar** [comment]
> There are many things in the spec which is not implemented by any server or clients ;-)

You are right. I will look at it and then decide. If HasNotifier would be a two way reference it would be easier.

**Comment by aldoshkind** [comment]
Hello @mar-ar @oroulet !
Can you please tell me if event propagation implemented at the moment? I am working on a project where this feature would be very handy.
